recipes/cdat_info/meta.yaml

```diff
@@ -0,0 +1,68 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "cdat_info" %}
+{% set version = "2.8" %}
+#{% set sha256 = "d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+    git_rev: v2.8
+    git_url: git://github.com/UV-CDAT/uvcdat.git
```

Comment:
I saw other people using `archive` directory under `github.com`, but did not find good documentation for it.  

I think that this is a very neat way to download the `github` package especially having the checksum.   If it saves time that is even better, this package is big and can take time to download on `travis`.  :wink: